checkers/regexpPattern_checker.go

```diff
@@ -0,0 +1,68 @@
+package checkers
+
+import (
+	"go/ast"
+	"go/constant"
+	"regexp"
+	"strings"
+
+	"github.com/go-lintpack/lintpack"
+	"github.com/go-lintpack/lintpack/astwalk"
+)
+
+func init() {
+	var info lintpack.CheckerInfo
+	info.Name = "regexpPattern"
+	info.Tags = []string{"diagnostic", "experimental"}
+	info.Summary = "Detects suspicious regexp patterns"
+	info.Before = "regexp.MustCompile(`google.com|yandex.ru`)"
+	info.After = "regexp.MustCompile(`google\\.com|yandex\\.ru`)"
+
+	collection.AddChecker(&info, func(ctx *lintpack.CheckerContext) lintpack.FileWalker {
+		domains := []string{
```

Comment:
Those domains might be at the end of the file.
BTW, do we need to extend them via checker param?

Reply:
It's a good idea to add custom domains support, but I think this list is good enough for the beginning.